src/vs/workbench/contrib/terminal/browser/media/shellIntegration.zsh

```diff
@@ -2,9 +2,23 @@
 #   Copyright (c) Microsoft Corporation. All rights reserved.
 #   Licensed under the MIT License. See License.txt in the project root for license information.
 # ---------------------------------------------------------------------------------------------
-
 autoload -Uz add-zsh-hook
 
+. ~/.zshenv
```

Comment:
I think this will error and maybe print output to the terminal if `~/.zshenv` doesn't exist?